Installation/cmake/modules/CGAL_add_test.cmake

```diff
@@ -43,6 +43,11 @@ if(ANDROID)
   set(ANDROID_DIR_PREFIX /data/local/tmp/)
   find_program(adb_executable adb)
 endif()
+if(SSH)
+  set(SSH_DIR_PREFIX /home/pi/CGAL/)
```

Comment:
I think that should be a cache variable, with a documentation (and a better name). Maybe unify with `ANDROID_DIR_PREFIX`, and rename it to `CGAL_REMOVE_TEST_DIR_PREFIX` (and the default value would be different on Android and rPI).